save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/test/suite/PluginsWithoutTests.kt

```diff
@@ -0,0 +1,33 @@
+package com.saveourtool.save.preprocessor.test.suite
+
+import com.saveourtool.save.test.TestSuiteValidationProgress
+import com.saveourtool.save.test.TestSuiteValidationResult
+import com.saveourtool.save.testsuite.TestSuiteDto
+import com.saveourtool.save.utils.getLogger
+
+/**
+ * Plug-ins without tests.
+ */
+@TestSuiteValidatorComponent
+class PluginsWithoutTests : AbstractTestSuiteValidator() {
+    override fun validate(
+        testSuites: List<TestSuiteDto>,
+        onStatusUpdate: (status: TestSuiteValidationResult) -> Unit,
+    ) {
+        require(testSuites.isNotEmpty())
```

Comment:
Same question

Reply:
Whether the list of test suites is empty or not is checked at higher level -- see `TestSuiteValidationService#validateAll()`, line 67.